CMakeLists.txt

```diff
@@ -113,6 +113,8 @@ endif()
 
 if( eckit_FOUND AND oops_FOUND AND ioda_FOUND AND bufr_FOUND )
     set(iodaconv_bufr_query_ENABLED True)
+    find_package( bufr_query REQUIRED )
```

Comment:
`bufr_query` I believe is specific to a few agencies and needs to be build separately outside of JEDI

@emilyhcliu maybe we can discuss in our tag-up, as know you're another user of `bufr_query`

Reply:
ok yes that obvious now looking at it

I'm definitely confused now though,, there was a `iodaconv_bufr_query` that was added to `bufr2ioda` and thought this one related to this,,, and then `bufr_query` was something GDAS-App related?



@srherbener or @emilyhcliu or @rmclaren may know more... we'll need this CMake to remain generic for all partners even those that do not use NCEP BUFR